peer.go

```diff
@@ -423,8 +424,23 @@ func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) error {
 		p.activeChanMtx.Lock()
 		p.activeChannels[chanID] = lnChan
 		p.activeChanMtx.Unlock()
+
+		activeChans = append(activeChans, *chanPoint)
 	}
 
+	// As a final measure we launch a goroutine that will ensure the
+	// channels are not currently disabled, as that will make us skip it
+	// during path finding.
+	go func() {
+		for _, chanPoint := range activeChans {
+			err := p.server.announceChanStatus(chanPoint, false)
```

Comment:
This'll cause us to continually announce a channel as enabled each time a peer reconnects. Something we've been trying to avoid. It would necessarily be _each_ time (due to the trickle timeout), but it'll maximize the number of channel updates in flight for flappy peers.

Reply:
Ah nvm, I see now that the method it uses will noop if we already announced this state.

Reply:
Would be nice to note that here in a comment.

Reply:
Added comment.